x-pack/solutions/observability/plugins/profiling/test/scout/api/tests/has_setup.spec.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, apiTest, test } from '@kbn/scout-oblt';
+import type { ScoutTestConfig } from '@kbn/scout-oblt';
+import { format as formatUrl } from 'url';
+import supertest from 'supertest';
+import { getRoutePaths } from '../../../../common';
+import { getProfilingPackagePolicyIds, deletePackagePolicy } from '../../common/utils/fleet';
+import {
+  cleanUpProfilingData,
+  loadProfilingData,
+  setupProfiling,
+} from '../../common/utils/profiling_data';
+
+const profilingRoutePaths = getRoutePaths();
+
+function getSupertest(config: ScoutTestConfig) {
+  const kibanaServer = config.metadata.config.servers.kibana;
+  const url = formatUrl({
+    ...kibanaServer,
+    auth: 'elastic:changeme',
```

Comment:
Hardcoding the credentials will prevent from running the tests on MKI. Maybe you can use `kbnClient` fixture directly for `cleanUpProfilingData` ?

Reply:
right, credentials are available on the `kbnClient`.